src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -541,7 +548,42 @@ SHAMapStoreImp::clearSql (DatabaseCon& database,
             std::this_thread::sleep_for (
                     std::chrono::milliseconds (setup_.backOff));
     }
-    journal_.debug << "finished: " << deleteQuery;
+    JLOG(journal_.debug) << "finished: " << deleteQuery;
+}
+
+void
+SHAMapStoreImp::clearSql(DatabaseCon& database,
```

Comment:
This is only called from one function: `clearCaches`. I'd remove this function and just put the code inline in `clearCaches`